launcher/image/test/test_ssh_manual.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -euxo pipefail
+
+print_usage() {
+    echo "usage: test_ssh_manual.sh [-i imageName] [-p imageProject]"
+    echo "  -i <imageName>: which image name to use for the VM"
+    echo "  -p <imageProject>: which image project to use for the VM"
+    exit 1
+}
+
+run_ssh_test() {
+    BUILD_ID=$(date +%s)
+    HOME_DIR=$(echo ~)
+    VM_NAME="cs-ssh-test-$BUILD_ID"
+    WORKLOAD_IMAGE='us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/basic-test:latest'
+    ZONE="us-central1-a"
+
+    ACCOUNT_NAME=$(gcloud config list account --format "value(core.account)" | tr @. _)
+    PROJECT_NAME=$(gcloud config get-value project)
+
+    # Create a new VM
+    source create_vm.sh -n $VM_NAME -i $IMAGE_NAME -p $IMAGE_PROJECT -m tee-image-reference=$WORKLOAD_IMAGE,tee-container-log-redirect=true,tee-cmd=["newCmd"],tee-env-ALLOWED_OVERRIDE=overridden,enable-osconfig=TRUE -z $ZONE
+
+    # Add an SSH public key to an OS Login profile
+    gcloud compute os-login ssh-keys add --key-file=$HOME_DIR/.ssh/google_compute_engine.pub || true
```

Comment:
How is the key generated?

Reply:
This key is one of the defaults that is created when connecting to GCE. Since this is a manual test, I figured it is likely this key exists for those who will be running it. The alternative here is to generate and delete a key specifically for this test, which I can implement if preferred?

Reply:
This key is one of the defaults that is created when connecting to GCE. Since this is a manual test, I figured it is likely this key exists for those who will be running it. The alternative here is to generate and delete a key specifically for this test, which I can implement if preferred?